tests/smartcam/test_smartcamdevice.py

```diff
@@ -23,6 +23,12 @@ async def test_state(dev: Device):
     assert dev.is_on is not state
 
 
+@device_smartcam
+async def test_device_id(dev: Device):
+    """Test all devices have a device id."""
```

Comment:
Is this relevant only for smartcams or should this test go to test_device?

Reply:
Yes makes sense